Parsers/ASimAuditEvent/Parsers/ASimAuditEventGithubWebhook.yaml

```diff
@@ -0,0 +1,81 @@
+Parser:
+  Title: ASIM Audit Event parser for GithubWebhook
+  Version: '0.1.0'
+  LastUpdated: Apr 30, 2024
+Product:
+  Name: GithubWebhook
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: GithubWebhook
+  Link: https://docs.github.com/en/webhooks/webhook-events-and-payloads
+Description: |
+  This ASIM parser supports normalizing the GithubWebhook logs to the ASIM Audit Event normalized schema.
+ParserName: ASimAuditEventGithubWebhook
+EquivalentBuiltInParser: _ASim_AuditEvent_GithubWebhook
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (
+    disabled:bool = false
+  )
+  {
+    let EventDetailslookup=datatable(EventOriginalType:string,EventResult:string, EventType:string, EventSeverity:string)
+     ["push","Success","Install","Informational",
+      "create","Success","Create","Informational",
+      "delete","Success","Delete","Informational",
+      "repository","Success","Other","Informational",
+      "pull_request","Success","Other","Informational",
+      "membership","Success","Other","Informational",
+      "member","Success","Other","Informational",
+      "fork","Success","Create","Informational",
+      "installation","Success","Create","Informational",
+      "team","Success","Create","Informational",
+      "organization","Success","Other","Informational",
+      "label","Success","Other","Informational"
```

Comment:
does github webhook provide these 12 event_s or are there more? if more, we should configure all.

Reply:
Configured 25 with reference to Product document